ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/ServerSentEvents.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.plugins.sse
+
+import io.ktor.client.*
+import io.ktor.client.engine.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.util.*
+import io.ktor.util.logging.*
+
+internal val LOGGER = KtorSimpleLogger("io.ktor.client.plugins.sse.ServerSentEvents")
+
+/**
+ * Indicates if a client engine supports Server-sent events.
+ */
+public object ServerSentEventsCapability : HttpClientEngineCapability<Unit> {
+    override fun toString(): String = "ServerSentEventsCapability"
+}
+
+/**
+ * Client Server-sent events plugin.
+ *
+ * @property reconnectionTimeMillis - time for client to wait before attempting to reconnect.
+ */
+public class ServerSentEvents(
```

Comment:
Why not to use new Client Plugin API?

Reply:
Thanks for comments, I will rewrite it with new Client Plugin API